knora-ontologies/knora-base.ttl

```diff
@@ -2703,11 +2920,99 @@ rdfs:label rdfs:subPropertyOf :objectCannotBeMarkedAsDeleted .
                                                "Representation (Text)"@en ,
                                                "Répresentation (Texte)"@fr ,
                                                "Rappresentazione (testo)"@it ;
-                    
+
                     rdfs:comment "A resource containing text files"@en .
 
 
 
+###  http://www.knora.org/ontology/knora-base#Person
+
+:Person rdf:type owl:Class ;
+
+        rdfs:subClassOf :Resource, foaf:Person,
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasFamilyName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasGivenName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasIAFIdentifier ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasBirthDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasDeathDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ];
+
+        rdfs:label "Person"@de ,
+                   "Person"@en ,
+                   "Personne"@fr ,
+                   "Persona"@it ;
+
+        rdfs:comment "A resource representing a person"@en .
+
+
+###  http://www.knora.org/ontology/knora-base#Publication
+
+:Publication rdf:type owl:Class ;
```

Comment:
> Meanwhile, I think it would make sense to create a separate bibliography ontology that could be considered less stable than knora-base.

This would be fine! Let's do it.